db/migrations/20221108152353_create-water-charge-versions.js

```diff
@@ -6,7 +6,7 @@ exports.up = async function (knex) {
   await knex
```

Comment:
Not strictly related to this PR but I noticed while reviewing that we have a lot of migrations where the `up` method is `await knex...` while the down method is `return knex...`. The [example here](https://knexjs.org/guide/migrations.html#transactions-in-migrations) implies that `return knex...` is the preferred way of doing it:
```js
exports.up = function(knex) {
  return knex.schema
    .createTable('users', function (table) {
        // ...
    })
};
```
So, do we want to change this? And if so, do we change it in this PR or a separate one? Alternatively, "if it ain't broke don't fix it" and we maybe just follow the convention going forwards?

Reply:
I think we should fix it for the same reason I gave in the PR; we tend to copy existing migrations that match what we are trying to do and use them as a template for the new one.

So, I might as well get it done in this PR as our team convention is to be consistent!